core/domain/user_services.py

```diff
@@ -2002,3 +2003,12 @@ def get_community_reviewer_usernames(review_category, language_code=None):
         raise Exception('Invalid review category: %s' % review_category)
 
     return get_usernames(reviewer_ids)
+
+
+def log_username_change(user_id, current_username, new_username):
```

Comment:
user_id is unclear -- is it the ID of who changed the model or the ID of the person whose name was changed? Please be clearer.

Reply:
Updated the comment.